packages/devextreme/testing/tests/DevExpress.ui.widgets/drawer.markup.tests.js

```diff
@@ -179,6 +179,61 @@ QUnit.module('rendering', () => {
         assert.ok($element.hasClass(DRAWER_CLASS + '-top'), 'top panel position class added');
     });
 
+    [true, false].forEach((rtlEnabled) => {
+        const panelPositions = ['top', 'bottom', 'left', 'right', 'before', 'after'];
+
+        const configs = [
+            { panelPosition: 'top', expectedClass: `${DRAWER_CLASS}-top` },
+            { panelPosition: 'bottom', expectedClass: `${DRAWER_CLASS}-bottom` },
+            { panelPosition: 'left', expectedClass: `${DRAWER_CLASS}-left` },
+            { panelPosition: 'right', expectedClass: `${DRAWER_CLASS}-right` },
+            { panelPosition: 'before', expectedClass: rtlEnabled ? `${DRAWER_CLASS}-right` : `${DRAWER_CLASS}-left` },
+            { panelPosition: 'after', expectedClass: rtlEnabled ? `${DRAWER_CLASS}-left` : `${DRAWER_CLASS}-right` },
+        ];
+
+        configs.forEach(({ panelPosition, expectedClass }) => {
+            QUnit.test(`drawer should have ${expectedClass} class on initialization if panel position is set to ${panelPosition}, rtlEnabled=${rtlEnabled}`, function(assert) {
+                assert.expect(6);
+
+                const $element = $('#contentTemplate').dxDrawer({
+                    position: panelPosition,
+                    opened: true,
+                    rtlEnabled,
+                });
+
+                assert.strictEqual($element.hasClass(expectedClass), true, `class ${expectedClass} is added correctly`);
+
+                panelPositions
+                    .filter((position) => `${DRAWER_CLASS}-${position}` !== expectedClass)
+                    .map((position) => {
```

Comment:
[minor]
it's more natively to use `forEach` if you just want to iterate. `map` is designed for mapping

Reply:
Thanks for your remark. I assume this is not critical for the tests and will take this into account in future development.